lib/src/execute.rs

```diff
@@ -302,6 +303,7 @@ impl ExecuteCtx {
         );
 
         info!("request completed in {:.0?}", request_duration);
+        info!("CPU Time {:.0?}", cpu_time);
```

Comment:
probably good to have a message more consistent with the other `info` text, something like...
```suggestion
        info!("request completed using {:.0?} of CPU time", cpu_time);
```

additionally moving this up a line or two would give output like:
```
request completed using ... of WebAssembly heap
request completed using ... of CPU time
request completed in ...
```
which leaves the first two grouped together as they're both talking about resource use for the request in question. that seems preferable to the `WebAssembly heap/request completed/CPU used` sandwich we'd get here?